apps/web/src/components/maps/base-map.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { useEffect } from "react";
+import { MapContainer, TileLayer, FeatureGroup, Polygon } from 'react-leaflet';
+import { EditControl } from 'react-leaflet-draw';
+
+import 'leaflet/dist/leaflet.css';
+import 'leaflet-draw/dist/leaflet.draw.css';
+import useSWR from "swr";
+
+export default () => {
+  const { data, isLoading } = useSWR("/api/openstad/api/project/1/area/2");
```

Comment:
Dit is nog hardcoded

Reply:
Dat klopt inderdaad bij de kaart, omdat we hier nog niet de functionaliteit hebben geschreven waarbij we het in een bepaalde pagina ophalen. Dit maakt voor nu gebruik van een hardcoded locatie, om te zorgen dat er in ieder geval altijd een werkend voorbeeld is voor dit proof of concept van de map. (Ik heb echter wel alle hardcoded referenties in de area-pagina's weggehaald!)

Reply:
Als het goed is, omdat areaId een foreign key is van project, dan zou je met het ophalen van het project de area ook op moeten kunnen halen als je dat in de route ook zo configureert. Zie includeConfig op widget route voor een voorbeeld. Dan is er ook minder kans op het doorgeven een verkeerde projectId en areaId combinatie.

Reply:
Dat klopt inderdaad, en dat zal ook zonder twijfel werken... Maar het voorbeeld van de kaart staat op een pagina die niet onder een project valt, namelijk op /localhost:31470/map .  Daardoor heeft hij niet van zichzelf een project. Ik kan het echter wel in een project zetten en dan vervolgens wel dat opnemen!